Blase/lean-toolchain

```diff
@@ -0,0 +1 @@
+leanprover/lean4:nightly-2025-08-11
```

Comment:
@bollu can you make this file a symlink to the root lean-toolchain file, so that they are kept in sync?

Would anything break if we do so?

Reply:
I think that would work just fine!